### PR TITLE
use `bt` (binary type) param if it exists

### PR DESCRIPTION
**Pull request by erwinmombay**
allows us to monitor other buckets

### CODE REVIEW

**Comment by erwinmombay** [comment]
@jridgewell can you point me to where i can add tests for these changes?

File: routes/error-tracker.js [hunk 1]
```diff
@@ -102,7 +103,11 @@ function handler(req, res) {
   } else {
     errorType = errorType + '-1p';
   }
-  if (canary) {
+
+  // Do not append binary type if 'prod' since that is the default
+  if (binaryType && binaryType !== 'prod') {
```
Comment (jridgewell) [inline routes/error-tracker.js]: Technically correct, but I'd rather the `binaryType !== 'prod'` be in a nested if statement.

Comment (erwinmombay) [reply routes/error-tracker.js]: done. PTAL

**Comment by erwinmombay** [comment]
@jridgewell PTAL

**Pull request merged**